apps/i18n/common/en_us.json

```diff
@@ -45,6 +45,8 @@
   "courses": "Courses",
   "coursesCardAction": "Find a course",
   "coursesCardDescription": "Find a new course to keep challenging yourself.",
+  "coursesHeadingText": "Courses",
```

Comment:
I added these strings thinking most would be changed, but this one in particular might not, so I will reuse the existing.